apps/mobile-wallet/src/App.tsx

```diff
@@ -93,31 +93,50 @@ const AppContent = () => {
   return (
     <Provider store={store}>
       <PersistQueryClientContextProvider createPersister={createTanstackAsyncStoragePersister}>
-        <ApiContextProvider>
```

Comment:
I moved this provider so that API requests fire only after the persisted cache has been restored.